docs/usage/agent-identity.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+parent: Usage
+---
+
+# Agent identity
+
+Zincati agent tries to derive a unique identity for the machine it is running on by introspecting the underlying OS and reading user configuration.
+This includes assigning an ID and a group label specific to the agent, so that cluster-wide upgrades can be orchestrated via [phased rollouts][phased] and [lock-based][fleetlock-strategy] reboots.
+
+[phased]: auto-updates.md#phased-rollouts-client-wariness-canaries
+[fleetlock-strategy]:  updates-strategy.md#lock-based-strategy
+
+## Identity configuration
+
+All agent identity values are normally auto-detected at startup and do not require user intervention.
+
+However, the following settings can be overridden through configuration fragments in the `identity` section:
+ * `group`: group label, used for graph fetching ([Cincinnati][cincinnati]) and reboot orchestration ([FleetLock][fleetlock])
+ * `node_uuid`: agent ID, used for graph fetching ([Cincinnati][cincinnati]) and reboot orchestration ([FleetLock][fleetlock])
+ * `rollout_wariness`: agent wariness to [phased rollouts][phased], used for graph fetching ([Cincinnati][cincinnati]).
+
+The following are defaults for each setting:
+- `group` (group label) is set to `default`
+- `node_uuid` (agent ID) is automatically derived from `/etc/machine-id` in an application-specific way
```

Comment:
When you say application-specific, who is the application here?

It would be useful if `node_uuid` exactly matched the machine id, since other systems like Kubernetes nodes use that as node metadata as well. Saying its "derived from" in an "application-specific" way seems pretty open ended.

Reply:
The application here is Zincati. The term "application-specific" is lifted from `sd_id128_get_machine_app_specific` [docs](https://www.freedesktop.org/software/systemd/man/sd_id128_get_machine.html#), which is the implementation used under the hood. I should probably add this reference directly in the docpage.

The machine-id is considered confidential and should not be exposed on the network, see https://www.freedesktop.org/software/systemd/man/machine-id.html.

Reply:
Fair enough. In future, if a protocol implementer needed to associate Zincati identify with nodes I suppose a different mechanism would be needed.

Reply:
The auto-generated default is meant to be enough to cover a scenario where the network flows are trusted and the node-id does not need to be cross-linked with other inventory systems.
Relaxing the former would need some kind of authN mechanism. 
For the latter, a fragment with `identity.node_uuid = <your_custom_id>` should already be enough to cover that.